dbms/src/Common/SpaceSaving.h

```diff
@@ -273,7 +280,14 @@ class SpaceSaving
             push(counter);
         }
 
-        for (size_t i = 0; i < nextAlphaSize(m_capacity); ++i)
+        readAlphaMap(rb);
```

Comment:
Now the bucket frequencies (alpha map) are serialised/deserialised as well which should increase precision in distributed tables (or when processing a lot of rows).